### PR TITLE
Remove travis

### PR DESCRIPTION
**Pull request by emcastillo**
Remove travis as we now use github actions for lint now

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -49,10 +49,6 @@
         'sphinx==3.0.4',
         'sphinx_rtd_theme',
     ],
-    'travis': [
-        '-r stylecheck',
-        '-r docs',
-    ],
```
Comment (toslunar) [inline setup.py:55]: It's used in `.github/workflows/pretest.yml`.
https://github.com/cupy/cupy/blob/436b09b1ed5255fae95a4e4f41d2875fa4f7fceb/.github/workflows/pretest.yml#L23

```diff
-        READTHEDOCS=True pip install -U -e .[travis]
+        READTHEDOCS=True pip install -U -e .[stylecheck,docs]
```

Comment (emcastillo) [reply setup.py:55]: 🙇‍♂️

**Comment by leofang** [comment]
Bye bye Travis! I will miss you, the once fastest CI.

**Review by toslunar — APPROVED** [review]

**Comment by toslunar** [comment]
pfnCI, test this please.

**Pull request merged**